controllers/apartments.rb

```diff
@@ -1,14 +1,23 @@
-# index
+# index route
 get '/apartments' do
+	@apartments = Apartment.all
   erb :"apartments/index"
 end
 
-# new
+# new route
 get '/apartments/new' do
   erb :"apartments/new"
 end
 
-# show
+# show route
 get '/apartments/:id' do
+	@apartment = Apartment.find(params[:id])
+	@tenants = Tenant.where(apartment_id: @apartment[:id])
```

Comment:
This is correct, but there's a shorter, more readable way using AR association helpers:
`@tenants = @apartment.tenants`.